pkg/rkenodeconfigclient/client.go

```diff
@@ -153,3 +184,58 @@ func getConfig(client *http.Client, url string, header http.Header) (*rkeworker.
 	nc := &rkeworker.NodeConfig{}
 	return nc, json.NewDecoder(resp.Body).Decode(nc)
 }
+
+// getKubeletCertificateFilesFromProcess finds the tls-private-key-file and tls-cert-file values from
+// the kubelet process so that they may be used to determine the validity of the kubelet certificates stored
+// on the host.
+func getKubeletCertificateFilesFromProcess(processes map[string]types.Process) (string, string) {
+	proc, ok := processes["kubelet"]
+	if !ok {
+		return "", ""
+	}
+
+	return findCommandValue("--tls-private-key-file", proc.Command), findCommandValue("--tls-cert-file", proc.Command)
```

Comment:
this is just a comment but I wonder if it's possible that a user defines their own kubelet serving cert and causes our logic to miss and constantly request a new certificate even though RKE is not necessarily generating the cert... shouldn't be a big problem (the user can self resolve this by having a valid cert) but still something that crossed my mind.